### DISCUSSION TITLE
Reading From Depth Camera

### DISCUSSION THREAD

**Discussion by utkudotdev** [empriselab/RCareWorld#61] [Help - Programming] [answered]
Hi! I am trying to now read from the camera, but unfortunately, there are errors at runtime. Here is the code I have used:
```python
camera = env.create_camera(
    id=654321, name="camera", width=1920, height=1116, fov=60, is_in_scene=True
)
camera.getRGB()
```
I got the width, height, and FOV values by looking at the output of `getCameraInfo`. I think I have done everything correctly. I checked the docs, but they appear to be outdated, as the initialization methods they talk about no longer seem to exist. This is the error I am getting:
```
Traceback (most recent call last):
  File "hackathon_demo/test_bathing.py", line 240, in <module>
    camera.getRGB()
  File "/home/utku/Code/RCareWorld/pyrcareworld/pyrcareworld/sensors/camera.py", line 74, in getRGB
    reshaped_image_rgb = image_rgb.reshape(self.height, self.width, 3)
ValueError: cannot reshape array of size 488792 into shape (1116,1920,3)
```

**Comment by YoruCathy** [comment, marked as answer]
Added two test scripts and updated the reshape stuff. Could you pull and see if it works?

**Reply by utkudotdev** [reply]
It does work, thank you!
